src/main/java/io/github/sefiraat/emctech/slimefun/groups/EmcSlimefunDictionaryFlexGroup.java

```diff
@@ -59,20 +60,21 @@ public boolean isVisible(Player player, PlayerProfile playerProfile, SlimefunGui
 
     @Override
     @ParametersAreNonnullByDefault
-    public void open(Player player, PlayerProfile profile, SlimefunGuideMode mode) {
+    public void open(Player p, PlayerProfile profile, SlimefunGuideMode mode) {
```

Comment:
```suggestion
    public void open(Player player, PlayerProfile profile, SlimefunGuideMode mode) {
```

Reply:
this was changed to `p` for consistency between `open` methods of flex groups